src/js/api/fetchMoviesTrending.js

```diff
@@ -23,27 +22,30 @@ const fetchMoviesTrending = () => {
       }
       return response.json();
     })
-    .then(data => {
-      const movieIds = data.results.map(movie => movie.id);
-      const promises = movieIds.map(id => fetchMovieGenres(id));
-      return Promise.all(promises).then(genres => {
-        return data.results.map((movie, index) => ({
-          ...movie,
-          genres: genres[index],
-        }));
-      });
+   .then(data => {
+      const movieData = Promise.all(
+        data.results.map(async movie => {
+          try {
+            const genres = await fetchMovieGenres(movie.id);
+            movie.genres = genres;
+            return movie;
+          } catch (error) {
+            console.error(error);
+            movie.genres = '';
+            return movie;
+          }
```

Comment:
Ok. Przydałoby się też info do usera korzystającego ze strony - Something went wrong, if you want to see genres please reload page.